packages/foundry/src/vault/adapter/aave/aaveV2/IAaveV2.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.15;
+
+import { IERC20Upgradeable as IERC20 } from "openzeppelin-contracts-upgradeable/token/ERC20/extensions/ERC4626Upgradeable.sol";
+
+interface IScaledBalanceToken {
+  /**
+   * @dev Returns the scaled balance of the user. The scaled balance is the sum of all the
+   * updated stored balance divided by the reserve's liquidity index at the moment of the update
+   * @param user The user whose balance is calculated
+   * @return The scaled balance of the user
+   **/
+  function scaledBalanceOf(address user) external view returns (uint256);
+}
+
+// Aave aToken (wrapped underlying)
+interface IAToken is IERC20, IScaledBalanceToken {
+  /**
+   * @dev Returns the address of the underlying asset of this aToken (E.g. WETH for aWETH)
+   **/
+  function UNDERLYING_ASSET_ADDRESS() external view returns (address);
+
+  /**
+   * @dev Returns the address of the incentives controller contract
+   **/
+  function getIncentivesController() external view returns (IAaveMining);
+
+  function POOL() external view returns (address);
+}
+
+// Aave liquidity mining interface
+interface IAaveMining {
+  function claimRewards(
+    address[] calldata assets,
+    uint256 amount,
+    address to
+  ) external returns (uint256);
+
+  /*
+   * @dev Returns the configuration of the distribution for a certain asset
+   * @param asset The address of the reference asset of the distribution
+   * @return The asset index, the emission per second and the last updated timestamp
+   **/
+  function getAssetData(address asset)
```

Comment:
This is the code i found from https://polygonscan.com/address/0x357d51124f59836ded84c8a1730d72b749d8bc23#readProxyContract
```
 function assets(address asset)
    external
    view
    returns (
      uint128,
      uint128,
      uint256
    );
    ```